man/rgblink.5

```diff
@@ -56,55 +40,105 @@ and
 .Ql \et .
 Other backslash escape sequences in
 .Xr rgbasm 5
-are only relevant to assembly code and do not apply in section names.
+are only relevant to assembly code and do not apply in linker scripts.
+.Ss Directives
+.Bl -tag -width Ds
+.It Including other files
+.Ql Ic INCLUDE Ar path
+acts as if the contents of the file at
+.Ar path
+were copy-pasted in place of the
+.Ic INCLUDE
+directive.
+.Ar path
+must be a string.
+.It Bank specification
```

Comment:
Since all the other directive headings are of the form "*Verb*ing the *noun*":
```suggestion
.It Specifying the current bank
```

Reply:
This specific wording is to introduce the name, since it's referred to elsewhere in the document.

Reply:
Was this comment meant in regard to the next review point, about keeping the quotes in the `Changing the "current address"` heading?

Reply:
And likewise, we could quote here, `Specifying the "active bank"`?

Reply:
I think we should quote both or neither. I'd prefer if neither were than both.

Reply:
> Was this comment meant in regard to the next review point, about keeping the quotes in the `Changing the "current address"` heading?

No, in regards to breaking the "verbing the noun" pattern.